src/Contracts/Pools/OpdexLiquidityPool.cs

```diff
@@ -226,6 +224,11 @@ protected void UpdateReserves(ulong balanceCrs, UInt256 balanceSrc)
         
         Log(new ReservesLog { ReserveCrs = balanceCrs, ReserveSrc = balanceSrc });
     }
+
+    protected void UpdateKLast()
```

Comment:
Consider possible gas savings by passing in `ReserveCrs`/`ReserveSrc`. Sacrifices brevity, but I'd say it's probably worth it since their values are retrieved from state several times in some methods.

Reply:
We can't exactly do this because there are some instances in the `LiquidityPool.cs` method during `Mint` or `Burn` where we store the local `ReserveCrs` and `ReserveSrc` in variables but when we update `KLast` those values have changed so we need to pull latest from state